features/protocollering/verblijfplaatshistorie/protocollering.feature

```diff
@@ -0,0 +1,66 @@
+      | pl_id |
+      | 1001  |
+      En de persoon is ingeschreven op adres 'A1' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+      | 0530                              | 20000101                           |
+      En de response van de downstream api heeft de volgende headers
+      | x-geleverde-pls |
+      | 1001            |
+    
+  
+  Regel: Gebruikte parameters worden vertaald naar rubrieknummers volgens Logisch ontwerp BRP en vastgelegd in het veld 'request_zoek_rubrieken'.
+
+    Scenario: Raadpleeg verblijfplaatshistorie met peildatum
+      Als verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde                |
+      | type                | RaadpleegMetPeildatum |
+      | burgerservicenummer | 000000012             |
+      | peildatum           | 2023-05-26            |
+      Dan heeft de persoon met burgerservicenummer '000000012' de volgende 'protocollering' gegevens
+      | request_zoek_rubrieken                 |
+      | 010120, 081030, 081320, 581030, 581320 |
+
+    Scenario: Raadpleeg verblijfplaatshistorie met peildatum
+      Als verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde              |
+      | type                | RaadpleegMetPeriode |
+      | burgerservicenummer | 000000012           |
+      | datumVan            | 2023-01-01          |
+      | datumTot            | 2024-01-01          |
+      Dan heeft de persoon met burgerservicenummer '000000012' de volgende 'protocollering' gegevens
+      | request_zoek_rubrieken                 |
+      | 010120, 081030, 081320, 581030, 581320 |
+
+  Regel: Velden van de response worden vertaald naar rubrieknummers volgens Logisch ontwerp BRP en vastgelegd in het veld 'request_gevraagde_rubrieken'.
+
+    Scenario: Raadpleeg verblijfplaatshistorie met peildatum
+      Als verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde                |
+      | type                | RaadpleegMetPeildatum |
+      | burgerservicenummer | 000000012             |
+      | peildatum           | 2023-05-26            |
+      Dan heeft de persoon met burgerservicenummer '000000012' de volgende 'protocollering' gegevens
+      | request_gevraagde_rubrieken                                                                                                                                                                                                                                                                                           |
+      | 10120, 080910, 081010, 081110, 081115, 081120, 081130, 081140, 081150, 081160, 081170, 081180, 081190, 081210, 081310, 081330, 081340, 081350, 580910, 581010, 581110, 581115, 581120, 581130, 581140, 581150, 581160, 581170, 581180, 581190, 581210, 581310, 581330, 581340, 581350, PAVP01, PAVP02, PAVP03, PAVP04 |
```

Comment:
10120 moet dat niet zijn 010120?
Ik denk dat ik iets mis. Waarom is dit lijstje zoveel langer dan het lijstje in het request?

Reply:
hoe bedoel je de vraag dat dit lijstje langer is? Dat gevraagde rubrieken meer zijn dan zoekrubrieken?
Gezocht wordt met burgerservicenummer, plus datum aanvang (actueel en historisch, NL of buitenland)
Gevraagd worden alle velden die in de response zitten.

10120 (burgerservicenummer - moet inderdaad 010120 zijn) hoort niet bij gevraagde rubrieken, want die zit niet in de response. Heb ik aangepast.